test/unit/collection-view.spec.js

```diff
@@ -1399,16 +1399,6 @@ describe('collection view', function() {
     });
   });
 
-  describe('has a valid inheritance chain back to Backbone.View', function() {
```

Comment:
I'm not sure what this was attempting to prove.. I think it was just a poorly maintained test.  `this.ChildView` is a `Backbone.View`